youtube_dl/extractor/openload.py

```diff
@@ -13,7 +13,7 @@
 
 
 class OpenloadIE(InfoExtractor):
-    _VALID_URL = r'https://openload.(?:co|io)/(?:f|embed)/(?P<id>[a-zA-Z0-9-_]+)'
+    _VALID_URL = r'https://openload\.(?:co|io)/(?:f|embed)/(?P<id>[a-zA-Z0-9-_]+)'
```

Comment:
Should `r'https://` be `r'https?://'? Or if the site is only in https, thats how it should be?